test_suite/Cargo.toml

```diff
@@ -11,6 +11,8 @@ license = "Apache-2.0"
 
 [dependencies]
 scale-info = { path = "..", features = ["derive", "serde"] }
+# TODO: this should work
```

Comment:
I believe @ascjones has a fix in the works for this.

Reply:
Yes I'm working on it

Reply:
https://github.com/paritytech/scale-info/pull/74